charmcraft.yaml

```diff
@@ -9,3 +9,7 @@ bases:
     run-on:
     - name: ubuntu
       channel: "22.04"
+parts:
+  charm:
+    charm-binary-python-packages:
+      - cryptography
```

Comment:
Have you checked if the charm can be packed? `tls_certificates` docs advise to add more sections to the file: https://github.com/canonical/manual-tls-certificates-operator/blob/f09693c834739b5911e8fac44ed9bfb08e9c9630/lib/charms/tls_certificates_interface/v2/tls_certificates.py#L21